app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,46 @@
+<%= stylesheet_link_tag "recipedetails" %>
+
+<div class="container">
+  <h1><%= @recipe.name %></h1>
+  <div class="details">
+    <p>Preparation Time: <%= @recipe.preparation_time %> minutes</p>
+    <p>Cooking Time: <%= @recipe.cooking_time %> minutes</p>
+    <p>Description: <%= @recipe.description %></p>
+    <p>Public: <%= @recipe.public ? 'Yes' : 'No' %></p>
+
+    <% if @recipe.user == current_user %>
+  <div class="button-group">
+    <%= link_to "Generate Shopping List (Coming Soon)", "#", class: "btn btn-primary disabled" %>
```

Comment:
- [ ] Kindly adhere to the project requirement, it seems like you didn't implement the `inventory shopping list` feature in your project, according to which you should use the shopping list button, which further opens a `popup` menu for creating the inventory shopping list. Please focus more on the requirement and implement things accordingly, don't forget to implement the feature as it looks in the wireframe image below.

### Requirement
![image](https://github.com/DevKaranJ/recipes/assets/66476329/22ba95fb-ba74-486e-9e71-8121effcb73e)

![image](https://github.com/DevKaranJ/recipes/assets/66476329/3a012d09-586e-4b9a-a932-939598886396)


- [ ] Please also provide a shopping list page in your application as it is given in the wireframe.

![inventory_shopping_list](https://github.com/DevKaranJ/recipes/assets/66476329/cfed3a92-f16c-46b0-9cc5-8e3c50eb409c)